app/javascript/packages/document-capture/components/document-capture.tsx

```diff
@@ -173,6 +179,8 @@ function DocumentCapture({ onStepChange = () => {} }: DocumentCaptureProps) {
             onStepSubmit={trackSubmitEvent}
             autoFocus={!!submissionError}
             titleFormat={`%{step} - ${appName}`}
+            skipDocAuth={skipDocAuth}
+            inPersonFirstStep={skipDocAuth ? steps[0].name : undefined}
```

Comment:
when inPersonFirstStep is undefined then we're starting off w/ document upload correct?

Reply:
inPersonFirstStep is being used inside form-steps. If skipDocAuth is true, we need to set the name of the step. Because we only need to do this for IPP, I just set the name to undefined. We would not hit the block to set the name. (The code in form-steps sets the url to #prepare.)

Reply:
ahhh ok thank you for explaining this!